data-plane/config/template/500-receiver.yaml

```diff
@@ -1,3 +1,19 @@
+---
```

Comment:
why all the noice w/ the headers ?

Reply:
Do you mean copyright headers or YAML document separators?
In the first case, in `eventing` we use them in all files, so I followed, in the second case to append one file after the other, is it not necessary?